### PR TITLE
Changed billing response model

### PR DESCRIPTION
**Pull request by sup2ch**
No billing P2P errors now.

### CODE REVIEW

File: aioqiwi/kassa/types/invoice.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Invoice(BaseModel):
     pay_url: str = Field(..., alias="payUrl")
     amount: Amount = Field(..., alias="amount")
     status: Status = Field(..., alias="status")
-    customer: Customer = Field(..., alias="customer")
+    customer: Optional[Customer] = Field(type(None), alias="customer")
```
Comment (uwinx) [inline aioqiwi/kassa/types/invoice.py:42]:
```suggestion
    customer: Optional[Customer] = Field(None, alias="customer")
```

`type(None)` is an old type generator's issue

**Comment by uwinx** [comment]
@sup2ch, shall we merge? are you sure you tested well?

**Comment by sup2ch** [comment]
I have been using the library with this fix for 3 days, everything is fine.

**Comment by uwinx** [comment]
Okay, will push it to master for now. Not gonna bump the version for this, install it from master

**Pull request merged**